svg-2-android-vector/src/main/java/com/quittle/svg2androidvector/PreBuildDependencyAdder.java

```diff
@@ -0,0 +1,27 @@
+package com.quittle.svg2androidvector;
+
+import com.android.build.gradle.api.BaseVariant;
+
+import org.gradle.api.Action;
+import org.gradle.api.Task;
+import org.jetbrains.annotations.NotNull;
+
+
```

Comment:
Unnecessary double newline

Reply:
gah dunno why the formatter didn't catch this.